client/android/WorldScope/app/src/main/java/com/litmus/worldscope/MainActivity.java

```diff
@@ -1,28 +1,52 @@
 package com.litmus.worldscope;
 
-import android.app.Activity;
+import android.content.Context;
 import android.content.Intent;
-import android.support.design.widget.TabLayout;
+import android.graphics.Bitmap;
+import android.graphics.BitmapShader;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.os.Bundle;
 import android.support.design.widget.FloatingActionButton;
-import android.support.design.widget.Snackbar;
-import android.support.v7.app.AppCompatActivity;
-import android.support.v7.widget.Toolbar;
-
+import android.support.design.widget.TabLayout;
 import android.support.v4.app.Fragment;
 import android.support.v4.app.FragmentManager;
 import android.support.v4.app.FragmentPagerAdapter;
 import android.support.v4.view.ViewPager;
-import android.os.Bundle;
-import android.view.LayoutInflater;
+import android.util.Log;
+import android.view.View;
```

Comment:
The import statements can be arranged. `import android.view.View;` can be moved downloads to join the other `android.view`